pkg/vm/vm.go

```diff
@@ -1556,11 +1556,9 @@ func (v *VM) throw(item stackitem.Item) {
 	v.handleException()
 }
 
-// Call calls method by offset. It pushes new context to the invocation stack
-// and increments invocation counter for the corresponding context script hash.
+// Call calls method by offset using new execution context.
 func (v *VM) Call(ctx *Context, offset int) {
 	v.call(ctx, offset)
-	v.Invocations[ctx.ScriptHash()]++
```

Comment:
We also use `(v *VM) Call` during contract witness verification, so I'd better add this logic to `(bc *Blockchain) InitVerificationVM` also, because contract verification script is not prohibited to use `System.Runtime.GetInvocationCounter`, and it may affect verification process.

Reply:
The problem here is that `InitVerificationVM` works with VM. And even bigger problem is that `GetTestVM` also returns a VM instead of interop.Context.

Reply:
I took a look at the C# code, and It behaves in the same manner (no increment), so maybe we can create an issue about that?

Reply:
Why do you think so? `VerifyWitness -> LoadContract -> LoadScript -> LoadContext` and there is `invocationCounter.TryAdd(state.ScriptHash, 1)` in `LoadContext`.

Reply:
Yes, you're right, I missed that.